Pod/Classes/Lifecycle/Behaviors/Nav-Bar-Hairline-Fade/NavBarHairlineFadeBehavior.swift

```diff
@@ -101,6 +108,12 @@ fileprivate extension NavBarHairlineFadeUpdater {
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
         let shift = scrollView.contentInset.top + scrollView.contentOffset.y
         hairlineAlpha = shift.scaled(from: contentOffsetFadeRange, to: 0...1, clamped: true)
+        
+        #if DEBUG
+        if logsContentOffset {
+            print("content y-offset: \(shift)")
```

Comment:
#nth log `"\(NavBarHairlineFadeUpdater)"` or `"\(NavBarHairlineFadeBehavior.self)"` somewhere in here, so if someone forgets to turn this off, they can easily find where it came from.

Reply:
#nth format the number with a max of 3 decimal places, or at least strip trailing zeros, because it's pretty ungainly otherwise. Don't cast to `Int`, though, because you might be throwing away information on 2x and 3x screens.